apps/web/src/pages/templates/components/templates-store/TemplatesStoreModal.tsx

```diff
@@ -164,7 +243,27 @@ export const TemplatesStoreModal = ({ isOpened, onClose }: ITemplatesStoreModalP
             </ActionIcon>
           </TemplateHeader>
           <CanvasHolder>
-            <div>Canvas</div>
+            <ReactFlowProvider>
+              <FlowEditor
+                steps={selectedTemplate.steps}
+                nodeTypes={nodeTypes}
+                zoomOnScroll={false}
+                zoomOnPinch={false}
+                zoomOnDoubleClick={false}
+                panOnDrag={false}
+                panOnScroll
+                preventScrolling={false}
+                nodesDraggable={false}
+                elementsSelectable={false}
+                nodesConnectable={false}
+                selectNodesOnDrag={false}
+                wrapperStyles={{
+                  height: '100%',
+                  width: '100%',
+                  minHeight: '300px',
+                }}
```

Comment:
reusing the same `FlowEditor` component as we do use in the Template Editor, allowing to pass any `react-flow` props